packages/ses/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ses",
-  "version": "0.12.7+1-dev",
+  "version": "0.13.0",
```

Comment:
✅ This is correct, but the changelog doesn’t reflect the reason: there were breaking changes. Perhaps this was missed in the format of a conventional commit?

Reply:
You added features.